src/tests/examples/output_compare_test.py

```diff
@@ -138,691 +138,710 @@ def check_output_values(study_path):
-    enable_study_output(study_path, False)
+    enable_study_synthesis(study_path, False)
     check_output_values(study_path)
     
 @pytest.mark.short
 @pytest.mark.skipif(sys.platform=="linux",
                     reason="Results different between linux and windows.")
 def test_unfeasible_problem_09(use_ortools, ortools_solver, solver_path):
     study_path = ALL_STUDIES_PATH / "specific-tests" / "unfeasible-problem" /"unfeasible_problem_09"
-    enable_study_output(study_path, True)    
+    enable_study_synthesis(study_path, True)    
     st = Study(str(study_path))
     st.set_variable(variable="include-unfeasible-problem-behavior", value="warning-dry", file_nick_name="general")
     run_study(solver_path, study_path, use_ortools, ortools_solver)
-    enable_study_output(study_path, False)
+    enable_study_synthesis(study_path, False)
     check_output_values(study_path)
     
 @pytest.mark.short
 @pytest.mark.skipif(sys.platform=="linux",
                     reason="Results different between linux and windows.")
 def test_unfeasible_problem_10(use_ortools, ortools_solver, solver_path):
     study_path = ALL_STUDIES_PATH / "specific-tests" / "unfeasible-problem" /"unfeasible_problem_10"
-    enable_study_output(study_path, True)    
+    enable_study_synthesis(study_path, True)    
     st = Study(str(study_path))
     st.set_variable(variable="include-unfeasible-problem-behavior", value="warning-dry", file_nick_name="general")
     run_study(solver_path, study_path, use_ortools, ortools_solver)
-    enable_study_output(study_path, False)
+    enable_study_synthesis(study_path, False)
     check_output_values(study_path)
 
 @pytest.mark.short
 def test_renewables_1(use_ortools, ortools_solver, solver_path):
     study_path = ALL_STUDIES_PATH / "short-tests" / "renewables-1"
-    enable_study_output(study_path, True)
+    enable_study_synthesis(study_path, True)
     run_study(solver_path, study_path, use_ortools, ortools_solver)
-    enable_study_output(study_path, False)
+    enable_study_synthesis(study_path, False)
     check_output_values(study_path)
 
 @pytest.mark.short
 def test_renewables_2(use_ortools, ortools_solver, solver_path):
     study_path = ALL_STUDIES_PATH / "short-tests" / "renewables-2"
-    enable_study_output(study_path, True)
+    enable_study_synthesis(study_path, True)
     run_study(solver_path, study_path, use_ortools, ortools_solver)
-    enable_study_output(study_path, False)
+    enable_study_synthesis(study_path, False)
     check_output_values(study_path)
 
 @pytest.mark.short
 def test_renewables_3(use_ortools, ortools_solver, solver_path):
     study_path = ALL_STUDIES_PATH / "short-tests" / "renewables-3"
-    enable_study_output(study_path, True)
+    enable_study_synthesis(study_path, True)
     run_study(solver_path, study_path, use_ortools, ortools_solver)
-    enable_study_output(study_path, False)
+    enable_study_synthesis(study_path, False)
     check_output_values(study_path)
```

Comment:
LGTM. Could you just update the function names for these tests like you did for the 3 new tests ?